.golangci.reference.yml

```
@@ -109,14 +109,34 @@ output:
   # Default: ""
   path-prefix: ""
 
-  # Sort results by: filepath, line and column.
+  # Sort results.
+  # See also `sort-order`.
```

We can improve the description:
```suggestion
  # Sort results by the order defined in `sort-order`.
```

---

It was my first idea, but I didn't find a good sentence.
Your proposal is good :+1: